tests/Test 2 - Add product to the cart.spec.js

```diff
@@ -0,0 +1,24 @@
+const {test, expect } = require('playwright/test')
```

Comment:
Same for file name + describe, login hook. Its even better to move this test to the same describe to reuse login actions.

Reply:
fixed